src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -63,68 +65,75 @@ class WorkflowActor private(binding: WdlBinding, actualInputs: Map[FullyQualifie
   private def notStarted: Receive = {
     LoggingReceive {
 
-      case Start(backend) =>
+      case Start(system, backend) =>
         _primeMover = Option(sender())
```

Comment:
This was more just me making sure I understood. We should discuss if we want the WorkflowManager when a single WF is run from the command line. My current htinking is that it is extra hassle for no gain